types/event.model.ts

```diff
@@ -0,0 +1,40 @@
+import { Crew, Team } from './crew.model';
+import { DisplayStatus } from './display-status.model';
+import { Round } from './round.model';
+
+export interface Event {
+	id: string;
+	regatta_id: string;
+	cost_id: string;
+	parent: null; // Wat is de 'parent'?
+	event: number;
+	number: number; // Is dit hetzelfde als 'event'?
+	day: Date; // Wordt een event op alleen een dag gedaan? Is dat anders dan de regatta?
+	code: string; 
+	name: string; // Is dit hetzelfde als 'name'?
+	remarks: string;
+	status: DisplayStatus; // Wat zijn de verschillende statussen?
+	category: string;
+	block_knrb: null; // Wat is dit?
+	boat_type: string; // Is dit een enum?
+	weighed: boolean;
+	sex: string;
+	correction_factor_boat: number;
+	sub_crews?: Crew[]; // Wat is het verschil tussen 'sub_crews' en 'crews'?
```

Comment:
Dit moet ik nakijken

Reply:
Stel je hebt:

V 1x en (VA 1x, VB 1x)

Dan is de crews van VA 1x leeg maar de sub_crews niet

Reply:
Dit is als ik het goed heb